tests/test_data/rewe.de/rewe.json

```diff
@@ -0,0 +1,42 @@
+{
+  "author": "Anne Lippert",
+  "canonical_url": "https://www.rewe.de/rezepte/wintergemuese-gnocchi-pfanne-kraeuterdip/",
+  "site_name": "Rewe.de",
```

Comment:
A small detail here: despite the meta `og:site_name` in the HTML, I think that `REWE` (all-caps) seems to be the preferred format for the website name.

If possible it would be nice to retrieve that from somewhere in the HTML -- or alternatively we could `raise StaticValueException(return_value='REWE')` from an override of the `site_name` method.

Reply:
I added a StaticValueException with the return_value='REWE' as you proposed since I was not able to get the correct format of the site_name from the HTML.